float64 left_torque
float64 right_torque